lib/realtime/rpc.ex

```diff
@@ -36,38 +36,29 @@ defmodule Realtime.Rpc do
   def enhanced_call(node, mod, func, args \\ [], opts \\ []) do
```

Comment:
wondering if it is clearer 
```suggestion
  def enhanced_call(node, mod, func, args \\ [], logger_ metadata \\ []) do
```

Reply:
actually, I don't see where these opts are applied to the logger, or am I missing something

Reply:
<img width="589" alt="Screenshot 2024-11-18 at 13 08 35" src="https://github.com/user-attachments/assets/c2b92137-a42a-4245-ab43-d016b97beccc">
used in on the rescue